src/components/explorer/IndividualPhenotypicFeatures.js

```diff
@@ -5,11 +5,8 @@ import { Icon, Table } from "antd";
 import { EM_DASH } from "../../constants";
 import { individualPropTypesShape } from "../../propTypes";
 import OntologyTerm from "./OntologyTerm";
-import { useIndividualResources } from "./utils";
 
 const IndividualPhenotypicFeatures = ({ individual }) => {
-    const resourcesTuple = useIndividualResources(individual);
-
     const columns = useMemo(() => [
```

Comment:
move this into a constant outside since it doesn't have any dependencies, like the other instances of columns constants